### PR TITLE
Compress celery payload

### PR DESCRIPTION
**Pull request by d-wysocki**
I want to merge this change because...I want to compress celery tasks to increase payload. Maximum payload before compressing `140.0 kB`, after changes `1.5 MB`.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3368 Too long message sent to celery.](https://app.clickup.com/t/2549495/SALEOR-3368)

**Review by patrys — COMMENTED** [review]
Would it make sense to set the [default compression method](https://docs.celeryproject.org/en/stable/userguide/configuration.html?highlight=task_compression#task-compression) instead?

**Review by korycins — COMMENTED** [review]
Why didn't you apply the compression to all email tasks? There is always a risk that someone will order a huge amount of different products and she/he will not get an email, because of this issue.

**Review by fowczarek — COMMENTED** [review]
You add file `dump.rdb`. Why do we need this file?

**Comment by d-wysocki** [comment]
@fowczarek That was my mistake. I deleted that file. Thanks.

**Review by maarcingebala — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ All notable, unreleased changes to this project will be documented in this file.
   - `channel` from `DraftOrderCreateInput` changed to channelId
   - `channel` from `DraftOrderInput` changed to channelId
   - `channel` from `pluginUpdate` changed to channelId
+- Compress celery tasks `send_fulfillment_confirmation_email_task`, `send_fulfillment_update_email_task`, `trigger_webhooks_for_event`, `send_webhook_request` - #7445 by d-wysocki
```
Comment (korycins) [inline CHANGELOG.md:113]: Changelog is not up-to-date. IMHO no need to explicitly list all tasks which received the compression. You can just add info that user_emails, webhooks etc.

**Review by fowczarek — APPROVED** [review]

**Comment by patrys** [comment]
Task linked: [SALEOR-3518 Error on saving long pages](https://app.clickup.com/t/2549495/SALEOR-3518)

**Comment by maarcingebala** [comment]
⬆️ Ignore this linking - my mistake.

**Pull request merged**